### ISSUE TITLE
Should show more NFT properties

### ISSUE THREAD

**Issue by seabornlee** [AlphaWallet/alpha-wallet-android#2683]
On iOS it shows `floor price`, `average price` and other properties, but Android doesn't show.

![IMG_5259](https://user-images.githubusercontent.com/735708/176127843-9d50913e-7332-4174-a958-4d721a14c41b.PNG)
![IMG_5258](https://user-images.githubusercontent.com/735708/176127858-b6b6cd1e-784b-4cee-b534-6722dd92b57e.PNG)

**Comment by hboon** [comment]
@seabornlee these might be relevant:

* https://github.com/AlphaWallet/alpha-wallet-ios/issues/4053
* https://github.com/AlphaWallet/alpha-wallet-ios/issues/3730

**Comment by JamesSmartCell** [comment]
@seabornlee these are shown where they're available. They are only available on Mainnet, Polygon and possibly Rinkeby.

Possibly the issue can be closed?

**Comment by seabornlee** [comment]
@JamesSmartCell I am comparing the same NFT on iOS and Android.

**Comment by JamesSmartCell** [comment]
@seabornlee here's what it looks like on Android:

![3e982260-b8f7-453c-a176-912efbb7315d](https://user-images.githubusercontent.com/12689544/176890176-692c472e-a2f8-445b-a4d5-3312cbb2bbf9.jpg)


Looking at it we don't have the floor price or average price. We had a chat about these a while back - 

@hboon with the floor price + last sale price/average price are these factored into the wallet value yet on iOS?

**Comment by hboon** [comment]
> @hboon with the floor price + last sale price/average price are these factored into the wallet value yet on iOS?

No, and I suppose with the Wallet API, we don't have to bother for now.

**Comment by hboon** [comment]
> Looking at it we don't have the floor price or average price. We had a chat about these a while back -

But we do display the floor private and average price on iOS though.

**Comment by JamesSmartCell** [comment]
Ok then, it looks like the action is to add floor price and average price.

@justindg can you add these two properties to the NFT Detail page?
